src/gilded_rose.spec.js

```diff
@@ -155,4 +156,21 @@ describe('generating ordered lists of items', () => {
     const testOutput = getHtmlListFromArray(list);
     expect(testOutput).toBe(`<ol><li>${JSON.stringify(sulfuras)}</li></ol>`);
   });
+  it('updates items and generates an ordered list with the updated items', () => {
+    const itemsList = [
```

Comment:
This looks good! Don't feel compelled to include _all_ of these as part of your tests in the future. Usually, when I'm doing tests like these that involve testing multiple objects or arrays, I max out that three or so. Nothing wrong with including them all here, but there's no need to test the number of passed in `items` as long as your use cases are covered.